src/runtime/dom-extras.ts

```diff
@@ -204,58 +204,56 @@ export const patchSlotInsertAdjacentElement = (HostElementPrototype: HTMLElement
   };
 };
 
+// TODO(STENCIL-1019): Evaluate if this patch should be modified
 /**
  * Patches the text content of an unnamed slotted node inside a scoped component
  * @param hostElementPrototype the `Element` to be patched
- * @param cmpMeta component runtime metadata used to determine if the component should be patched or not
  */
-export const patchTextContent = (hostElementPrototype: HTMLElement, cmpMeta: d.ComponentRuntimeMeta): void => {
-  if (BUILD.scoped && cmpMeta.$flags$ & CMP_FLAGS.scopedCssEncapsulation) {
-    const descriptor = Object.getOwnPropertyDescriptor(Node.prototype, 'textContent');
+export const patchTextContent = (hostElementPrototype: HTMLElement): void => {
```

Comment:
Looks like a lot of changes here, but it should all just be formatting from removing the top-level `if` condition